playwright/tests/refreshing-balance.spec.ts

```diff
@@ -32,3 +32,14 @@ test('Account selector should refresh balances on network change', async ({ page
   await expect(page.getByTestId('account-choice')).toContainText('456.0')
   await page.getByRole('button', { name: /Select/ }).click()
 })
+
+test('Accounts page should continuously refresh balance', async ({ page }) => {
+  await page.getByTestId('account-selector').click()
+  await expect(page.getByTestId('account-balance-summary')).toContainText('123.0')
+  await expect(page.getByTestId('account-choice')).toContainText('123.0')
+  await mockApi(page, 456)
+  await page.waitForRequest('**/chain/account/info/*', { timeout: 60_000 })
```

Comment:
This is a slow test. ACCOUNT_REFETCHING_INTERVAL is 30 seconds. Not sure if we should change it for tests

Reply:
I changed the interval based on NODE_ENV, but suspiciously test still fails, exceeding timeout

Reply:
Changed to REACT_APP_E2E_TEST